docs/uninstall-an-operator.md

```diff
@@ -0,0 +1,43 @@
+# How to uninstall an Operator?
+
+In order to uninstall an operator, you need to delete the following resources:
+
+- Subscription
+- ClusterServiceVersion (CSV)
+
+Both `Subscription` and `CSV` are namespaced objects meaning you need to delete a `Subscription` and a `CSV` in a specific namespace where you install the operator into.
+
+## Delete a Subscription
+
+If you wish to look up a list of `Subscription` in a specific namespace to see which `Subscription` you want to delete, you can use the following `kubectl` command:
+```bash
+$ kubectl get subscription -n <namespace>
+```
+
+You can delete the `Subscription` in the namespace that the operator was installed into using this command:
+```bash
+$ kubectl delete subscription <subscription-name> -n <namespace>
+```
+
+Alternatively, you can delete both `Subscription` and its `CSV` using a sequence of commands:
+```bash
+$ CSV=kubectl delete subscription <subscription-name> -n <namespace> -o json | jq '.status.installedCSV'
+$ kubectl delete subscription <subscription-name> -n <namespace>
+$ kubectl delete csv $CSV -n <namespace>
```

Comment:
This looks odd given that we follow this section with notes on how to delete a CSV. Should we add a section that covers deleting both object or update the existing section to cover `Deleting Subscriptions and CSVs`?

Reply:
Good point. I just move this part to the end of the doc. That way user will have to go through "Delete a CSV" part before reaching how to delete them both. Either way, it will have to involve at least 2 commands to delete a `Subscription` and a `CSV`.